src/replication.c

```diff
@@ -2956,7 +2925,7 @@ int slaveTryPartialResynchronization(connection *conn, int read_reply) {
         server.master_initial_offset = -1;
 
         
-        if (isOngoingRdbChannelSync()) {
+        if (server.repl_rdb_conn_state != REPL_RDB_CONN_STATE_NONE) { // rdb-channel sync is in progress
```

Comment:
if there's one thing that redis source code is consistent about is that for some reason it avoids using line comments.
in nearly anything else, it's inconsistent.